Library/PledgeStateCTAType.swift

```diff
@@ -50,7 +50,12 @@ public enum PledgeStateCTAType {
   public var titleLabel: String? {
     switch self {
     case .fix:
-      return Strings.Check_your_payment_details()
+      return localizedString(
```

Comment:
We already have these strings living in other PR. Will replace for the final keys once that PR is merged.